R/utils.R

```diff
@@ -92,7 +92,8 @@ sync_remote_file <- function(full_url, output_file, ...) {
 #' @return An SQL data frame
 #' @keywords internal
 read_parquet <- function(conn, path){
-    from_clause <- glue("FROM read_parquet('{path}')") |> sql()
+    path_list <- paste0("'", path, "'") |> paste(collapse = ", ")
+    from_clause <- glue("FROM read_parquet([{path_list}], union_by_name = true)")  |> sql()
```

Comment:
I only just discovered this, but I think [`glue_sql`](https://glue.tidyverse.org/reference/glue_sql.html) makes more sense here. It automatically adds the quotes and commas, and it's safer for SQL and you don't need to call `sql()` on the result.